tests/console/cpio.pm

```diff
@@ -0,0 +1,68 @@
+#SUSE"s openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Package: cpio
+# Summary: Create and extract archives with cpio, including all the supported archive formats
+# Maintainer: QE-Core <qe-core@suse.de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use Utils::Architectures;
```

Comment:
`use Utils::Architectures;` is not needed ?

Reply:
right, it is removed now